plugins/katello/3.13/advanced/http_proxies.md

```diff
@@ -0,0 +1,137 @@
+
+HTTP proxies can be created and then assigned to a product though bulk selection as well as for each individual repository. Additionally, Katello provides HTTP proxy policies for products or repositories. Policies include:
+  * Using the global HTTP proxy (the default)
+  * Using a specified HTTP proxy other than the global HTTP proxy
+  * Not using an HTTP proxy
+
+## Creation
+
+There are two ways of creating a HTTP Proxy for use in Katello: through a rake task or the Foreman UI.
+
+### Creating an HTTP Proxy with Rake
+
+```
+foreman-rake katello:update_default_http_proxy -- --name "A Great HTTP Proxy" --url "http://192.0.2.1:8888" --user admin --password redhat
+```
+
+### Creating an HTTP Proxy through the web UI
+
+  - navigate to: Infrastructure > HTTP Proxies
+  - click **New HTTP Proxy**
+
+![Creating a new HTTP proxy in the Foreman UI](/plugins/katello/{{ page.version }}/advanced/http_proxy_create_ui.png)
+
+- *Name*: This required option is used to identify the HTTP proxy.
+- *Url*: This required option is the URL of the proxy. Note that the scheme should be included. For example: "http://proxy.example.org:8888"
+- *Username*: This option is used for proxy authentication, if required.
+- *Password*: This option is used for proxy authentication, if required.
+
+The provided field for **Test Connection** may be used to verify the proxy fields are set correctly. The field accepts a URL that a GET request will be sent to via the proxy configured in the form. If successful you will see a user notification such as:
+
+
+![Test connect success message.](/plugins/katello/{{ page.version }}/advanced/test_connect_success.png)
+
+If there is a problem with the proxy configuration, you will see an error notification similar to:
+
+![Test connect failure message.](/plugins/katello/{{ page.version }}/advanced/test_connect_failure.png)
+
+## Removal
+
+To remove a HTTP Proxy:
+
+  - navigate to: Infrastructure > HTTP Proxies
+  - click **Delete** in the row of the proxy you want to remove
+
+## Setting the Global Default Content Proxy
+
+If an HTTP Proxy exists you can specify a global default HTTP proxy. By default newly created repositories will use this proxy for content synchronization requests. Existing repositories using the global default HTTP proxy policy will also be updated if the global default HTTP proxy setting changes.
+
+To set the global default:
+
+  - navigate to: Administer > Settings
+  - click the **Content** tab
+  - click the edit icon in the row labelled **Default http proxy** and select the proxy you wish to use as the default.
+
+![Settings Content tab.](/plugins/katello/{{ page.version }}/advanced/settings_content.png)
+
+Note that selecting different HTTP proxies will trigger updates to repositories that have the HTTP proxy policy set to the global default. Remote proxy settings will be updated asynchronously in a task.
```

Comment:
i would also be hesitant to recommend setting it via the setting while the installer is still controlling it

Reply:
So, just pull that from the docs?

Reply:
What about the use case where instead of creating a default a user wishes to change the default to another proxy? Should they re-run the installer? Or is changing the setting directly recommended?

Reply:
for now re-run the installer, since thats what everyone is used to doing.

Reply:
ok!